pages/garden/index.js

```diff
@@ -336,28 +373,48 @@ function Garden({ petCollection, onInteractPet }) {
             {currentPet.picture}
           </PetDisplay>
         </PetWrapper>
-        <ListPageLink>
-          <Link href="/pet-list">List</Link>
-          {/* update link to list page once replaced as mainpage required */}
-        </ListPageLink>
+        <AdjustedListPageLink>
+          <Link
+            href="/pet-list"
+            role="img"
+            aria-label="Staple of Books indicating List"
+          >
+            📚
+          </Link>
+        </AdjustedListPageLink>
+        <AdjustedDetailPageLink>
+          <Link
+            href={{
+              pathname: `/pet-details/${currentPet.id}`,
+              query: {
+                health: currentPet.status.health,
+                happiness: currentPet.status.happiness,
+                hunger: currentPet.status.hunger,
+                energy: currentPet.status.energy,
+                intelligence: currentPet.status.intelligence,
+              },
```

Comment:
To be honest I'm still unclear on the use of `query` in this context and why passing the pet's health, happiness, hunger, energy, and intelligence as query parameters when we're already navigating to the pet's details page using the pet's id in the URL?  

But perhaps we can take a closer look at that again.  ✨